frontend/src/index.css

```diff
@@ -1,3 +1,7 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
 :root {
```

Comment:
Please remove all the codes inside of the css files that we aren't using and remove the `App.css` file.